internal/buffer/out_message.go

```diff
@@ -69,25 +53,15 @@ func (m *OutMessage) OutHeader() *fusekernel.OutHeader {
 // insufficient space, it returns nil.
 func (m *OutMessage) Grow(n int) unsafe.Pointer {
 	p := m.GrowNoZero(n)
```

Comment:
Since there is no difference anymore between `Grow` and `GrowNoZero`, let’s unify them into one single method.

It would also be good to rename the method for clarity, since there is no growing going on anymore. Perhaps `AllocateAndAppend`?

Reply:
Ok, merged, I think Grow is fine, because it still grows the message